ee/server/settings/SAML.ts

```diff
@@ -0,0 +1,140 @@
+import { defaultAuthnContext, defaultAuthnContextTemplate, defaultAuthRequestTemplate, defaultLogoutResponseTemplate, defaultLogoutRequestTemplate, defaultMetadataCertificateTemplate, defaultMetadataTemplate, defaultNameIDTemplate, defaultIdentifierFormat } from '../../../app/meteor-accounts-saml/server/lib/constants';
+import { onLicense } from '../../app/license/server';
+import { settings } from '../../../app/settings/server/functions/settings';
+
+Meteor.startup(() => onLicense('SAML-enterprise', () => {
+	// role sync
+	settings.add('SAML_Custom_Custom_Default_role_attribute_sync', false, {
+		type: 'boolean',
+		group: 'SAML',
+		section: 'SAML_Section_4_Roles',
+		i18nLabel: 'SAML_Role_Attribute_Sync',
+		i18nDescription: 'SAML_Role_Attribute_Sync_Description',
+	});
+	settings.add('SAML_Custom_Custom_Default_role_attribute_name', '', {
+		type: 'string',
+		group: 'SAML',
+		section: 'SAML_Section_4_Roles',
+		i18nLabel: 'SAML_Role_Attribute_Name',
+		i18nDescription: 'SAML_Role_Attribute_Name_Description',
+	});
+	// advanced
+	settings.add('SAML_Custom_Custom_Default_allowed_clock_drift', false, {
+		type: 'int',
+		group: 'SAML',
+		enterprise: true,
+		invalidValue: false,
+		section: 'SAML_Section_6_Advanced',
+		i18nLabel: 'SAML_Allowed_Clock_Drift',
+		i18nDescription: 'SAML_Allowed_Clock_Drift_Description',
+	});
+	settings.add('SAML_Custom_Custom_Default_identifier_format', defaultIdentifierFormat, {
+		type: 'string',
+		group: 'SAML',
+		enterprise: true,
+		invalidValue: false,
```

Comment:
Change the invalid value of the template settings to be the same as the default values